wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,89 @@
+  - development
+  - osu!dev
+---
+
 # Discord servers
 
-Index page for all articles regarding community [Discord](https://discord.com/) servers.
+As the community grows and expands, several [Discord](https://discord.com/) servers have been made and maintained by the community to specialize in the game's many different facets.
+
+Each server provides a space with a specific purpose in mind, where players get together to discuss and work on various different projects. This index aims to serve as an index where users can find a subcommunity they can best fit in.
+
+## Community
+
+These are servers made by the community, for the community. They do not have a specific purpose. However, they serve as community hubs for users to interact in and be up to date with different aspects of the game.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [osu!](https://discord.gg/ppy) | [peppy](https://osu.ppy.sh/users/2) | The **osu!** Discord server, previously known as **osu!dev**, is osu!'s official Discord server for development-related purposes. The server functions as a hub for people to discuss and work on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the osu! team. |
+| [Aiess Project](https://discord.gg/2XV5dcW) | [Naxess](https://osu.ppy.sh/users/8129817) | The **Aiess Project** is in charge of maintaining and updating several feeds, informing users of the many different events that happen in osu! on a daily basis! Users can keep up with the latest qualified and soon-to-be ranked beatmaps, keep up with usergroup changes, and be up to date with the latest news posts. |
+
+### Project Loved
+
+<!-- This section as a whole needs to be discussed. Last time someone suggested listing the Loved suite of servers in the Project Loved wiki article, not everyone was on board. Moreover, some of its members and pepy themselves are not really happy on how everything is split apart. Question is whether we want to promote these servers while they exist, or ignore them in favour of the #osu-loved channel in the dev server -->
+
+[Project Loved](/wiki/Community/Project_Loved) organizes its workflows in different servers, divided by the ruleset which they specialize in. **The official development of the project is done in the osu! Discord server.** These hubs aim to facilitate the discussion of the mode's picks to be added. Non-game mode dependant discussions should always be carried out in `#osu-loved`, in the **osu!** Discord server.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu! Project Loved](https://discord.gg/gn58Uk5sTE) | [Librarian](https://osu.ppy.sh/users/10083084) |
+| [osu!catch Project Loved](https://discord.gg/phgtyS4UCh) | [Wesley](https://osu.ppy.sh/users/2407265) |
+| [osu!mania Loved Community](https://discord.gg/Ededv7m) | [Paturages](https://osu.ppy.sh/users/1375479) |
+| [Project Loved: Taiko](https://discord.gg/GhfjtZ6) | [Hivie](https://osu.ppy.sh/users/14102976) |
```

Comment:
sort by gamemode probably

Reply:
osu! -> mania -> taiko -> catch (Popularity)

or

catch -> mania -> osu! -> taiko (Alphabetic)

or

osu! -> taiko -> catch -> mania (Website icons)

Reply:
osu taiko catch mania is what i meant by that